addons/hr/views/hr_employee_views.xml

```diff
@@ -303,7 +303,7 @@
             <field name="name">hr.employee.list</field>
             <field name="model">hr.employee</field>
             <field name="arch" type="xml">
-                <list string="Employees" multi_edit="1" editable="bottom" sample="1" js_class="hr_employee_list">
+                <list string="Employees" multi_edit="1" sample="1" js_class="hr_employee_list">
```

Comment:
Why not just removing the create action instead of disabling the edition in that list view?

Reply:
It is a bit annoying for the user to edit the billable time target, then. 🤔 

(If it is that list view we used in the new menu inside Timesheets app)

The goal of the list view in the new menu you added in timesheets app is to let the timesheets manager to edit the billable time target. It is for that reason, we first display the list view because the list view be editable to quickly edit the billable time target for many employees. 🙂 

I would prefer disabling the creation or altering the behavior of the new button when the list view is editable to open the form view directly.

Reply:
okay, let's disable the creation for now.

Reply:
Alternatively, I think we just have to override this method to only call `await this.props.createRecord()` inside that method.

https://github.com/odoo/odoo/blob/fd4e8ea35eb2d722a133849afe7e9fe8e3cf3ac6/addons/web/static/src/views/list/list_controller.js#L267-L281